lib/python/picongpu/pypicongpu/species/operation/densityprofile/gaussian.py

```diff
@@ -0,0 +1,53 @@
+"""
+This file is part of the PIConGPU.
+Copyright 2024 PIConGPU contributors
+Authors: Brian Edward Marre, Afshari Masoud
+License: GPLv3+
+"""
+
+from .densityprofile import DensityProfile
+from .... import util
+from typeguard import typechecked
+
+
+@typechecked
+class Gaussian(DensityProfile):
+    """
+    Directional guassian density profile
+    """
```

Comment:
we should add the density formula here
```suggestion
    """
    gaussian density profile
    
    density( y < gasCenterFront ) = exp(gasFactor * (abs( (y - gasCenterFront) / gasSigmaFront))^gasPower)
    density( gasCenterRear < y ) = exp(gasFactor * (abs( (y - gasCenterRear) / gasSigmaRear))^gasPower)
    density( gasCenterFront >= y >= gasCenterRear ) = 1
    """
```